include/db.h

```diff
@@ -2324,6 +2324,30 @@ void register_object(dbref player, dbref location, const char *propdir, char *na
  */
 dbref remove_first(dbref first, dbref what);
 
+/**
+ * Determines if a given player is unable to (re)set the given flag
+ *
+ * The 'business logic' here is actually fairly dense and not easily
+ * summed up in a comment.  As this is a 'private' function, the
```

Comment:
It's a silly point, but this is no longer a private function, so probably remove the "As this is a private function," part of the comment :D